backend/hitas/models/property_manager.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from hitas.models._base import HitasModel
+from hitas.models.codes import PostalCode
+
+
+# Isännöitsijä
+class PropertyManager(HitasModel):
+    name = models.CharField(max_length=1024)
+    email = models.EmailField()
+    street_address = models.CharField(max_length=1024)
+    postal_code = models.ForeignKey(PostalCode, on_delete=models.PROTECT)
```

Comment:
Can you add the Address model back, and move all necessary fields there:
`email, street, postal code, city, phone number`
Then link all fields that use addresses to that